src/parser.js

```diff
@@ -459,6 +459,17 @@ export default class Parser extends Stream {
               this.manifest.discontinuityStarts.push(uris.length);
             },
             'program-date-time'() {
+              if (typeof this.manifest.dateTimeString === 'undefined') {
```

Comment:
Removed this in the last PR since we were going to do a major. But should probably wait it out till we do a proper major in VHS just to be safe.